android/src/oldarch/java/com/reactnativestripesdk/NativeStripeSdkModuleSpec.java

```diff
@@ -54,74 +54,6 @@ private void invoke(String eventName) {
     invoke(eventName, null);
   }
 
-  protected final void emitOnConfirmHandlerCallback(ReadableMap value) {
```

Comment:
Could we get these changes to NativeStripeSdkModuleSpec.java merged in a separate PR to `master`? It'll make it easier to merge the beta branch later.